src/main/java/dev/j3fftw/litexpansion/Events.java

```diff
@@ -68,6 +79,37 @@ public void onHungerDamage(EntityDamageEvent e) {
         }
     }
 
+    @EventHandler
+    @SuppressWarnings("ConstantConditions")
+    public void onGlassCut(PlayerInteractEvent e) {
+        final Block block = e.getClickedBlock();
+        if (block == null) return;
+
+        final Material blockType = block.getType();
+        final Location blockLocation = block.getLocation();
+        final GlassCutter glassCutter = (GlassCutter) SlimefunItem.getByID(Items.GLASS_CUTTER.getItemId());
+
+        if ((blockType == Material.GLASS
+            || blockType == Material.GLASS_PANE
+            || blockType.name().endsWith("_GLASS")
+            || blockType.name().endsWith("_GLASS_PANE")
+        ) && glassCutter.isItem(e.getItem())
+            && SlimefunPlugin.getProtectionManager().hasPermission(e.getPlayer(), blockLocation, ProtectableAction.BREAK_BLOCK)
+        ) {
+            e.setCancelled(true);
+
+            final SlimefunItem slimefunItem = BlockStorage.check(block);
+
+            if (slimefunItem == null && ((Rechargeable) SlimefunItem.getByItem(e.getItem()))
+                .removeItemCharge(e.getItem(), 0.5F)) {
+                blockLocation.getWorld().dropItemNaturally(blockLocation,
+                    new ItemStack(blockType));
+                block.setType(Material.AIR);
+                e.getPlayer().playSound(block.getLocation(), Sound.BLOCK_GLASS_HIT, 1.5F, 1F);
```

Comment:
Could be a SoundCategory.BLOCKS set here

Reply:
@J3fftw1 sounds are up to you

Reply:
Just add ,SoundCatergory.BLOCKS